src/panels/lovelace/components/hui-warning.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  CSSResult,
+  css,
+  customElement,
+} from "lit-element";
+
+@customElement("hui-warning")
+export class HuiWarning extends LitElement {
+  public getCardSize(): number {
```

Comment:
I don't think that this is needed? We also don't define a `setConfig`.

Reply:
Yeah, I questioned that and should have asked. Thanks.